### PR TITLE
{vis}[foss/2022a] Cellpose v2.2.2 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by deniskristak**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by deniskristak** [comment]
Requirements:
- https://github.com/easybuilders/easybuild-easyconfigs/pull/18003 
- https://github.com/easybuilders/easybuild-easyconfigs/pull/18002

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3159.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/fcfd50357087bdc92df0cea51aff61a7 for a full test report.

File: easybuild/easyconfigs/c/cellpose/cellpose-2.2.2-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        'preinstallopts': "export PBR_VERSION=1.2.3 && ",
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/seung-lab/fastremap/archive/'],
+        'checksums': ['ee0e9e798e9f331b7c46e98768735a42bb6e287b3aa56c31c322ba585f4036b7'],
+    }),
+    ('roifile', '2023.5.12', {
+        'checksums': ['32eeba0d9ad52cc249d6a234b737c1808a6c5d7d9baae6453709eb74222b3433'],
+    }),
+    (name, version, {
+        'preinstallopts': "sed -i 's/opencv-python-headless//g' setup.py && ",
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/c/cellpose/cellpose-2.2.2-foss-2022a.eb:49]: It's actually `opencv-contrib-python`, but makes sense, I'll use that instead.

```suggestion
        # OpenCV dependency provides opencv-python-headless, but it's not exposes as such to pip,
        # so strip out that requirement to make 'pip check' pass
        'preinstallopts': "sed -i 's/opencv-python-headless/opencv-contrib-python/g' setup.py && ",

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3159.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/7860c7cf2e8a78b93230bcc8401118d1 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3143.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/cc2578456e3de31e00628e9d96bd523e for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @deniskristak!

**Pull request merged**